mindsdb/interfaces/storage/model_fs.py

```diff
@@ -69,12 +69,12 @@ def file_get(self, name):
     def file_set(self, name, content):
         self.fileStorage.file_set(name, content)
 
-    def folder_get(self, name, update=True):
```

Comment:
update=False is used for big HF models. It allows to not download if it is already downloaded to instance

Reply:
- storage already has `sync` parameter to control copy behavior
 - local and s3 storages checks changes in files before upload/download even if sync is True
 - `folder_get` does not work for s3 storage